line_profiler/line_profiler.py

```diff
@@ -9,14 +9,14 @@
 from argparse import ArgumentError, ArgumentParser
 
 try:
-    from ._line_profiler import LineProfiler as CLineProfiler
+    from _line_profiler import LineProfiler as CLineProfiler
```

Comment:
It errors when trying to run kernprof if it has the relative import:
```py
>>> kernprof -lvz kern.py

Traceback (most recent call last):
  File "/home/theel/.pyenv/versions/3.9.5/lib/python3.9/site-packages/line_profiler/line_profiler.py", line 12, in <module>
    from ._line_profiler import LineProfiler as CLineProfiler
ModuleNotFoundError: No module named 'line_profiler._line_profiler'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/theel/.pyenv/versions/3.9.5/bin/kernprof", line 8, in <module>
    sys.exit(main())
  File "/home/theel/.pyenv/versions/3.9.5/lib/python3.9/site-packages/kernprof.py", line 238, in main
    import line_profiler
  File "/home/theel/.pyenv/versions/3.9.5/lib/python3.9/site-packages/line_profiler/__init__.py", line 15, in <module>
    from .line_profiler import __version__
  File "/home/theel/.pyenv/versions/3.9.5/lib/python3.9/site-packages/line_profiler/line_profiler.py", line 14, in <module>
    raise ImportError(
ImportError: The line_profiler._line_profiler c-extension is not importable. Has it been compiled? Underlying error is ex=ModuleNotFoundError("No module named 'line_profiler._line_profiler'")
```

Reply:
I think this is because you are writing the compiled module to `_line_profiler` and not `line_profiler._line_profiler` (The binary should be a submodule of the parent `line_profiler` module). 

I think if you rebase on the latest main and change the cython-extension method I was using to be the default instead of scikit-build, then it will work.

Reply:
Ohhhhh, that's quite possible. It's 11:26pm for me, so I'll try and have that fixed by the end of this week (I have a busy class week).